README-permission.md

```diff
@@ -98,7 +120,7 @@ Example playbook to make sure permission "MyPermission" is absent:
 
 ```yaml
 ---
-- name: Playbook to manage IPA permission.
+- name: Playbook to handle IPA permissions
```

Comment:
It is fine that all playbooks have the same title, but in this case, it is better to have named tasks, showing what is to be achieved. Also, named tasks are "good practices" detected by ansible-lint, so our examples would look better that way.

Reply:
Fixed